.github/workflows/test-install.yml

```diff
@@ -2,7 +2,7 @@ name: test-install
 
 on:
   push:
-    branch:
```

Comment:
Do we even need to limit the scope for this workflow?

Reply:
Yes, this is a very expensive workflow. Which is why we only run it for those branches (to protect them), and nightly.

Reply:
I see what you mean now. You are saying it might be sufficient to trigger on those paths, right?

Reply:
Yes. If someone creates a new branch, why shouldn't it run on that branch if it changes the requirements?

Reply:
Ok, I'll change that.